OrbitCore/TracepointEventBuffer.h

```diff
@@ -17,8 +17,8 @@ class TracepointEventBuffer {
  public:
   TracepointEventBuffer() : max_time_(0), min_time_(std::numeric_limits<uint64_t>::max()) {}
 
-  void AddTracepointEventAndMapToThreads(uint64_t time, uint64_t tracepoint_hash,
-                                         int32_t thread_id);
+  void AddTracepointEventAndMapToThreads(uint64_t time, uint64_t tracepoint_hash, int32_t thread_id,
+                                         int32_t process_id, bool is_same_pid_as_target);
```

Comment:
Have the `process_id` parameter before the `thread_id` parameter as that's always the order in which these two fields are handled when together.